src/overlays/actors/ovl_Bg_Jya_Megami/z_bg_jya_megami.c

```diff
@@ -9,7 +9,15 @@ void BgJyaMegami_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaMegami_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaMegami_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void BgJyaMegami_SetupDetectLight(BgJyaMegami* this);
+void BgJyaMegami_DetectLight(BgJyaMegami* this, GlobalContext* globalContext);
+void BgJyaMegami_SetupExplosion(BgJyaMegami* this);
+void BgJyaMegami_Explosion(BgJyaMegami* this, GlobalContext* globalContext);
+
+extern u32 D_06005780;
+extern u32 D_06005C4C;
+extern u32 D_0600B9F8;
```

Comment:
D_06005780 is a dlist, type Gfx[]
D_06005C4C is a dynapoly collision header, but it should be UNK_TYPE while we wait for bgcheck stuff
D_0600B9F8 is what i mentioned in a previous comment, imo you cant say for sure what it is yet unless you had a look at the object file or something, so UNK_TYPE